lib/package.rb

```diff
@@ -35,9 +35,10 @@ class << self
     attr_accessor :name, :cached_build, :in_build, :build_from_source, :in_upgrade
   end
 
-  def self.load_package(pkg_file, pkg_name = File.basename(pkg_file, '.rb'))
+  def self.load_package(pkg_file)
```

Comment:
After analyzing this a bit more, I think passing the full path to the file is fine in this case since it is only used internally.